### PR TITLE
Make seatservice-example container testable

### PR DESCRIPTION
**Pull request by vasilvas99**
- Update seatservice container to version v0.3.0 and clean-up manifest
- Update databroker to v0.3.0
- Add sdv-ctr-exec utility script to easily execute stripts inside containers
- Update recipes for container manifest deployment

### CODE REVIEW

**Review by stlachev — APPROVED** [review]
LGTM

File: meta-leda-components/recipes-sdv/sdv-wifi/iwd-config_git.bb [hunk 1]
```diff
@@ -12,16 +12,22 @@
...
-    install -m 0644 ${THISDIR}/files/main.conf ${D}/etc/iwd
+    install -m 0644 ${WORKDIR}/main.conf ${D}/etc/iwd
 }
 
+SRC_URI += "file://LICENSE"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
 PACKAGES = "${PN}"
 FILES:${PN} += "/etc/iwd/main.conf"
```
Comment (mikehaller) [inline meta-leda-components/recipes-sdv/sdv-wifi/iwd-config_git.bb:32]: I think this should be using `${SYSCONFDIR}`
Same for the install command in L23-24

File: meta-leda-distro/recipes-sdv-distro/images/sdv-image-full.bb [hunk 2]
```diff
@@ -23,7 +23,9 @@ IMAGE_INSTALL:append = " packagegroup-sdv-core"
 IMAGE_INSTALL:append = " packagegroup-sdv-additions"
 IMAGE_INSTALL:append = " packagegroup-sdv-tools"
 IMAGE_INSTALL:append = " packagegroup-sdv-examples"
-IMAGE_INSTALL:append:raspberrypi4-64 = " packagegroup-sdv-rpi4wifi"
+IMAGE_INSTALL:append = " ${@bb.utils.contains("DISTRO_FEATURES", "wifi", "packagegroup-base-wifi", "", d)}"
```
Comment (mikehaller) [inline meta-leda-distro/recipes-sdv-distro/images/sdv-image-full.bb:26]: Reusing the existing `wifi`DISTRO_FEATURE looks good now.

**Review by mikehaller — CHANGES_REQUESTED** [review]
Just the use of the SYSCONFDIR should be changed, everything else looks okay to me.

**Comment by vasilvas99** [comment]
Closed since the default behavior of seatservice-example should  be virtual can and bridge networking.

**Pull request closed without merging**